app/grandchallenge/algorithms/urls.py

```diff
@@ -104,6 +105,11 @@
     path("<slug>/jobs/", JobsList.as_view(), name="job-list"),
     path("<slug>/jobs/create/", JobCreate.as_view(), name="job-create"),
     path("<slug>/jobs/<uuid:pk>/", JobDetail.as_view(), name="job-detail"),
+    path(
+        "<slug>/jobs/<uuid:pk>/status-badge",
```

Comment:
URLs should always end with a `/`

```suggestion
        "<slug>/jobs/<uuid:pk>/status-badge/",
```